src/contexts/ConfigurationContext.tsx

```diff
@@ -431,10 +439,12 @@ export const ConfigContextProvider = ({ children }: Props) => {
           config_summary_default_language as SummaryLanguage,
           "auto"
         ),
+        defaultStyle: config_summary_default_style ?? "default",
```

Comment:
If we refer to the same value in multiple places then I suggest we extract it into a variable that can be referenced in multiple places. In this case I see "default" reference on line 273 and 442. If we extract this into a variables called `DEFAULT_STYLE` then it will make the code easier to maintain and we reduce the risk of someone accidentally changing this value in one place but not the other.

Reply:
Agree. Updated and also done for DEFAULT_LANG (instead of "auto")